client/coral-admin/src/components/App.css

```diff
@@ -0,0 +1,9 @@
+html, body, #root, #root > div {
```

Comment:
`#root` will be transformed by _CSS modules_. To keep all global you need to wrap everything `:global { .. }`

Reply:
But `#root` is defined outside the react context is it not?

Reply:
That's why you need to do this :-)